src/FizzWare.NBuilder/Generators/GetRandom.cs

```diff
@@ -546,6 +551,7 @@ public static string State()
         /// <summary>
         /// Generate a random e-mail address on a .com domain.
         /// </summary>
+        [Obsolete("Use Generators.Email.Random() instead.")]
         public static string Email()
         {
             return string.Format("{0}@{1}.com", String(8), String(7));

```

Comment:
For all of these now being marked `Obsolete`, how do you feel about just going ahead and making these pass-through methods? Update them to now call the corresponding `Generators.*.Random()`.